CONTRIBUTING.md

```diff
@@ -27,6 +27,15 @@ and using it in another project. Make sure that `mavenLocal` is added to the res
 sbt publishM2
 ```
 
+You can find a simple example application that uses Akka Serialization Helper [here](examples/example-app).
```

Comment:
I'd call the folder e.g. `akka-cluster-app` for clarity, `example` is kinda obvious since we're in `examples/`